docs/fsharp/get-started/get-started-vscode.md

```diff
@@ -42,58 +42,50 @@ You must also install the [.NET Core SDK](https://www.microsoft.com/net/download
+### [macOS](#tab/macos)
 
-![](media/getting-started-vscode/vscode-new-proj.png)
+Next, open the Command Palette (⌘+Shift+P) and type the following:
 
-> [!NOTE]
-If you don't see the above, try refreshing templates by running the following command in the Command Palette: `>F#: Refresh Project Templates`.
+>F# new project
 
-Select "F#: New Project" by hitting **Enter**, which will take you to this step:
+### [Linux](#tab/linux)
 
-![](media/getting-started-vscode/vscode-proj-type.png)
+Next, open the Command Palette (Ctrl+Shift+P) and type the following:
 
-This will select a template for a specific type of project.  There are quite a few options here, such as an [FsLab](https://fslab.org) template for Data Science or [Suave](https://suave.io) template for Web Programming.  This article uses the `classlib` template, so highlight that and hit **Enter**.  You will then reach the following step:
+>F# new project
 
-![](media/getting-started-vscode/vscode-new-dir.png)
+---
+
+This is powered by the [FORGE](https://github.com/fsharp-editing/Forge) project. You should see something like this:
 
-This lets you create the project in a different directory, if you like.  Leave it blank for now.  It will create the project in the current directory.  Once you press **Enter**, you will reach the following step:
+> [!NOTE]
+If you don't see template options, try refreshing templates by running the following command in the Command Palette: `>F#: Refresh Project Templates`.
 
-![](media/getting-started-vscode/vscode-proj-name.png)
+Select "F#: New Project" by hitting **Enter**, which will take you to the next step, which allows you to select project types.
 
-This lets you name your project.  F# uses [Pascal case](http://c2.com/cgi/wiki?PascalCase) for project names.  This article uses `ClassLibraryDemo` as the name.  Once you've entered the name you want for your project, hit **Enter**.
+Pick the `classlib` template and hit **Enter**.
 
-If you followed the previous step steps, you should get the Visual Studio Code Workspace on the left-hand side to look something like this:
+Next, pick a directory to create the project in. If you leave it blank, it will use the current directory.
 
-![](media/getting-started-vscode/vscode-new-proj-explorer.png)
+Finally, name your project in the final step. F# uses [Pascal case](http://c2.com/cgi/wiki?PascalCase) for project names. This article uses `ClassLibraryDemo` as the name. Once you've entered the name you want for your project, hit **Enter**.
 
-This template generates a few things you'll find useful:
+If you followed the previous step steps, you should get the Visual Studio Code Workspace on the left-hand side to appear with the following:
 
 1. The F# project itself, underneath the `ClassLibraryDemo` folder.
 2. The correct directory structure for adding packages via [`Paket`](https://fsprojects.github.io/Paket/).
```

Comment:
step 4: The `paket.exe` executable which -> The `paket.exe` executable that. Same change on steps 3 and 4 of the next section